tests/test-cases/Group6-VIC-Machine/6-12-Version.robot

```diff
@@ -0,0 +1,23 @@
+*** Settings ***
+Documentation  Test 6-12 - Verify vic-machine version command
+Resource  ../../resources/Util.robot
+
+*** Test Cases ***
+VIC-machine - Version check
+    Set Test Environment Variables
+
+    ${output}=  Run  bin/vic-machine-linux version
+    @{gotVersion}=  Split String  ${output}  ${SPACE}
+    ${version}=  Remove String  @{gotVersion}[2]
+    Log To Console  VIC machine version: ${version}
+    
+    ${result}=  Run  git rev-parse master
+    @{gotVersion}=  Split String  ${result}  ${SPACE}
+    ${commithash}=  Remove String  @{gotVersion}[0]
+    
+    Log To Console  Last commit hash from git: ${commithash}
+    
+    ${hash_result} =    Get Substring    ${version}    -7
```

Comment:
and I would move this up to where you were parsing the version above instead of splitting it with the git stuff in the middle